be/src/fs/fs_hdfs.cpp

```diff
@@ -3,25 +3,112 @@
 #include "fs/fs_hdfs.h"
 
 #include <fmt/format.h>
-#include <hdfs/hdfs.h>
```

Comment:
Better to create a new filesystem?

Reply:
Yes, we can abstract this interface to new filesystem.